### ISSUE TITLE
Fix all materials with normals - Log error while parsing material files

### ISSUE THREAD

**Issue by hwoarangmy** [OpenDungeons/OpenDungeons#529]
I guess it comes from @Danimal696 normal activation. I have logs:
Compiler error: unknown error in TrainingDummy1.material(23): token "ambient" is not recognized
Compiler error: unknown error in TrainingDummy1.material(24): token "diffuse" is not recognized
Compiler error: unknown error in TrainingDummy1.material(25): token "specular" is not recognized
Compiler error: unknown error in TrainingDummy1.material(26): token "emissive" is not recognized

I have that on many materials (StoneCoffin, TrainingDummy2, TrainingDummy3, TrainingDummy4, Treasury, Trollbed, ...)

If it is not a problem of my graphic card not supporting something, I guess they could be removed from material files safely.

**Comment by Danimal696** [comment]
That is strange, those values are used to tweak how much ligth is received, deflected or emitted. Works well on my end, erasing them would make the models look weird.

**Comment by oyvindln** [comment]
They are put in the wrong place.

**Comment by Danimal696** [comment]
then what would be the correct one?

**Comment by oyvindln** [comment]
You have to put the lighting values in the pass they are supposed to apply, not inside the texture unit. I'm not sure if they should go under the normal map pass or the texture pass, you will have to test.

```
// Treasury genrated by blender2ogre 0.6.0
import RTSS/NormalMapping_MultiPass from "RTShaderSystem.material"
material Treasury : RTSS/NormalMapping_MultiPass
{
    receive_shadows on

    technique
    {
        pass lighting
        {

             // Override the normal map.
             rtshader_system
             {
                lighting_stage normal_map Treasury_normal.png object_space 0 bilinear 1 -1.0
             }
        }

        pass decal
        {
            ambient 0.8 0.8 0.8 1.0
            diffuse 0.65 0.65 0.65 1.0
            specular 0.4 0.4 0.4 1.0 12.5
            emissive 0.0 0.0 0.0 1.0

            texture_unit decalmap
            {
                texture Treasury.png
                tex_address_mode wrap
                scale 1.0 1.0
                colour_op modulate
            }
        }
    }
}
```

**Comment by Danimal696** [comment]
Axctually is hwoarangmy who will have to, please tell me when you get the rigth place and ill fix them

**Comment by oyvindln** [comment]
You should be able to do that too, try to put them in one of the passes and which one makes the material look like you want. They should be generating errors for everyone. They are for me.

**Comment by Danimal696** [comment]
How do you see the error log? I read you (Oln) fixed the materials so that is one of the reasons they are not working now. Ill try when i have some free time

**Comment by oyvindln** [comment]
On windows I think it is in the appdata folder for you user.

**Comment by hwoarangmy** [comment]
@Danimal696 It should look something like:
C:\Users\Danimal\AppData\Roaming\opendungeons\
There, you wil find a file "opendungeons.log"

BTW, that's also where you will find replays and savegames. That will be usefull if you get a bug and you want to show it.

**Comment by oyvindln** [comment]
Will you have a look at this @Danimal696 ? Otherwise I'll sort it.
I think you have to put the lighting values under the lighting pass to have any effects. 

```
                tex_address_mode wrap
                scale 1.0 1.0
                colour_op modulate
```

these are default values I think, so they can probably be removed (at least the last two)

**Comment by Danimal696** [comment]
> Otherwise I'll sort it.

Please, Please, Please, Please, Please... with the little free time i have lately i prefer to make new content, and being honest i hate meddling with material files.

**Comment by Bertram25** [comment]
Fixed by #546
